Asterisk.2013/Asterisk.NET/Properties/AssemblyInfo.cs

```diff
@@ -1,20 +0,0 @@
-﻿using System.Reflection;
-using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
-
-[assembly: AssemblyTitle("AsterNET")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("AsterNET")]
-[assembly: AssemblyCopyright("Copyright © 2017")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
-[assembly: Guid("abe98502-ea83-4b04-98c3-ffe3eabe06b0")]
-[assembly: AssemblyVersion("1.3.0.0")]
-[assembly: AssemblyFileVersion("1.3.0.0")]

```

Comment:
Why is the AssemblyInfo.cs file being deleted?
Is this information saved elsewhere now?

Reply:
moved to project file